stellargraph/mapper/sampled_link_generators.py

```diff
@@ -125,10 +131,15 @@ def flow(self, link_ids, targets=None, shuffle=False):
                 src, dst = link
                 try:
                     node_type_src = self.graph.node_type(src)
+                except KeyError:
+                    raise KeyError(
+                        f"Node ID {src} supplied to generator not found in graph"
+                    )
+                try:
                     node_type_dst = self.graph.node_type(dst)
                 except KeyError:
                     raise KeyError(
-                        f"Node ID {n} supplied to generator not found in graph"
+                        f"Node ID {dst} supplied to generator not found in graph"
```

Comment:
It's just a minor tweak in the error messaging - previously if there was an `dst` node not found in graph it would print:

`f"Node ID {n} supplied to generator not found in graph"`

Reply:
I see what you mean. Yeah I just noticed it but didn't think it was big enough for its own PR so I fixed it here.

Reply:
I don't think any change is too small for a separate PR. For instance, by being in a separate PR this good improvement could've landed immediately (easy to review, easy to merge) rather than getting caught up in the review of the rest of this feature.

There's definitely some overhead of switching things around to be able to make a new PR. One good way to minimise this is to have multiple checkouts of the repo, so you can be working on a big feature in one (or more) directory, and do little changes in another without having to `git stash` or commit and then switch branches, and then switch back. Feel free to ask me if you want some tips about what has worked well for me.